rtos/rtx/TARGET_CORTEX_M/cmsis_os.h

```diff
@@ -88,8 +88,13 @@
 #if defined(TARGET_XDOT_L151CC)
 #define DEFAULT_STACK_SIZE         (WORDS_STACK_SIZE/2)
 #else
+#if defined (MBED_CFG_DEBUG_OPTIONS_COVERAGE)
+/* GCOV works with stack size approx. 32K. */
+#define DEFAULT_STACK_SIZE         (WORDS_STACK_SIZE*4*8)
```

Comment:
- Where is the 32k figure coming from? Is it experimental, or documented somewhere? That's quite a bit of stack.
- Does this mean that *each* thread that has some code compiled with coverage enabled needs to have at least 32k of stack size for coverage to work properly?